apps/web/vibes/soul/primitives/calendar.tsx

```diff
@@ -0,0 +1,48 @@
+import { ComponentPropsWithoutRef } from 'react'
+import { DayPicker } from 'react-day-picker'
+
+import clsx from 'clsx'
+import { ChevronLeftIcon } from 'lucide-react'
+
+const components = {
+  Chevron: () => <ChevronLeftIcon className="h-5 w-5" strokeWidth={1} />,
+}
+
+export function Calendar({
+  className,
+  classNames,
```

Comment:
This prop is now unused. Instead of spreading all props we might want to only expose a few? As it stands right now if `classNames` is passed, TypeScript won't complain but it will be ignored.